react/AppSections/Readme.md

```diff
@@ -0,0 +1,13 @@
+```
+const mockApps = require('./_mockApps').default;
+const Sections = require('./Sections').default;
+const { I18n } = require('../I18n')
+const locale = {};
+const handleAppClick = app => {
+    alert(JSON.stringify(app, null, 2))
+};
+
+<I18n dictRequire={lang => locale}>
+    <Sections apps={mockApps} t={x => x} onAppClick={handleAppClick} />
```

Comment:
Why do we need to pass it an instance of `t`?

Reply:
Fixed, there was no need.